File: js/palindrome.js
exports.palindrome = function(word) {
    originalWord = word.toString().toLowerCase().replace(/[^a-zA-z0-9]/g, "").split('').join('')
    reversedWord = word.toString().split('').reverse().join('').toLowerCase().replace(/[^a-zA-z0-9]/g, "")

    return originalWord == reversedWord ? true : false
};

File: python/palindrome.py
import re
def palindrome(word):
    word_to_string = str(word)
    original_word = re.sub(r'[^a-zA-Z0-9]', "", word_to_string.lower())
    reversed_word = re.sub(r'[^a-zA-Z0-9]', "", word_to_string[::-1].lower())

    return original_word == reversed_word




